README.md

```diff
@@ -33,108 +34,242 @@ Menu
+
+These tags don't show in the simple summary at the end of a k6 test (unless you reference them in a [threshold](#checks-and-thresholds)), but they are invaluable for filtering and investigating k6 test results if you use any of the [outputs](#outputs) mentioned below. k6 also supports simple hierarchical groups for easier code and result organization. You can find more information about groups and system and user-defined tags [here](https://docs.k6.io/docs/tags-and-groups).
+
+### Checks and thresholds
+
+Checks and thresholds are some of the k6 features that make it very easy to use load tests like unit and functional tests and integrate them in a CI (continuous integration) workflow.
+
+[Checks](https://docs.k6.io/docs/checks) are like asserts, but differ in that they don't halt execution, instead they just store the result of the check, pass or fail, and let the script execution continue. Checks are great for codifying assertions relating to HTTP requests/responses, making sure the response code is 2xx for example.
+
+[Thresholds](https://docs.k6.io/docs/thresholds) are global pass/fail criteria that can be used to verify any result metrics and make sure they stay in the desired bounds. They can even reference a subset of values in a given metric, based on the used metric tags. Thresholds are specified in the options section of a k6 script and if they are triggered during a test run, k6 would exit with a nonzero code, and optionally abort the test early. This makes thresholds ideally suited as checks in a CI workflow!
+
+### Example code
+
+```js
+console.log("TODO");
+```
+
+You can find (and contribute!) more k6 script examples here: [https://github.com/loadimpact/k6/tree/master/samples](https://github.com/loadimpact/k6/tree/master/samples)
+
+### Outputs
+
+To make full use of your test results and to be able to fully explore and understand them, k6 can output the raw metrics to a an external repository of your choice.
+
+The simplest output option, meant primarily for debugging, is to send the JSON-encoded metrics to a file or to `stdout`. Other ouput options are sending the metrics to an InfluxDB instance or to an Apache Kafka queue, or even to the Load Impact cloud. That way you can run your load tests locally or behind a company firewall, early in the development process or as a part of a CI suite, while at the same time being able store their results in the Load Impact cloud, where you can use Insights for comparison and analysis. You can find more information about the available outputs [here](https://docs.k6.io/docs/results-output) and about Load Impact Insights [here](https://docs.k6.io/docs/load-impact-insights) and [here](https://loadimpact.com/insights/).
```

Comment:
The simplest output option, meant primarily for debugging, is to send the JSON-encoded metrics to a file or to `stdout`. Other output options are sending the metrics to an InfluxDB instance, an Apache Kafka queue, or even to the Load Impact cloud. This allows you to run your load tests locally or behind a company firewall, early in the development process or as a part of a CI suite, while at the same time being able store their results in the Load Impact cloud, where you can use Insights for comparison and analysis. You can find more information about the available outputs [here](https://docs.k6.io/docs/results-output) and about Load Impact Insights [here](https://docs.k6.io/docs/load-impact-insights) and [here](https://loadimpact.com/insights/).